hopp/utilities/utilities.py

```diff
@@ -25,3 +25,29 @@ def load_yaml(filename, loader=Loader) -> dict:
         return filename  # filename already yaml dict
     with open(filename) as fid:
         return yaml.load(fid, loader)
+
+def check_create_folder(filepath):
+    if not os.path.isdir(filepath):
+        os.makedirs(filepath,exist_ok=True)
+        already_exists = False
+    else:
+        already_exists = True
+
+    return already_exists
+
+def write_yaml(filename,data):
+    if not '.yaml' in filename:
+        filename = filename +'.yaml'
+
+    with open(filename, 'w+') as file:
+        yaml.dump(data, file,sort_keys=False,encoding = None,default_flow_style=False)
+    return filename
```

Comment:
Is there a reason why a file name is being returned? It doesn't seem to be being modified, so it feels redundant.